test/test_congress.py

```diff
@@ -0,0 +1,19 @@
+import pytest
+from congress import Congress
+
+@pytest.fixture
+def congress():
+    api_key = 'testing_key'
+    return Congress(api_key)
+
+def test_all_members_getter(congress):
```

Comment:
This test in some sense tests both the propublica api and our handling of the response to the api.  For a unit test, we should assume the api is returning information as it should be and checking that our parsing functions and such are behaving as we expect.  You can use the `responses` package to mock out the response.